.gitignore

```diff
@@ -1,8 +1,17 @@
 # Files to be ignored
 
 # Eclipse
-.classpath
-.project
+# These 2 are remaining committed to ease configuration of the eclipse projects for the whole group

```

Comment:
can't we just delete this? And explain why they are deleted in the commit

Reply:
I don't mind doing it later, but I wanted to make sure everyone read it beforehand and not do the same mistake I did before